configure.ac

```diff
@@ -765,49 +766,29 @@ enable_llvm_default=no
 
 INVARIANT_AOT_OPTIONS=nimt-trampolines=900,ntrampolines=8000,nrgctx-fetch-trampolines=256,ngsharedvt-trampolines=2800
 
+with_bitcode_default=no
+with_cooperative_gc_default=no
+with_mobile_default=no
+with_mobile_static_default=no
+with_monodroid_default=no
+with_monotouch_default=no
+with_monotouch_tv_default=no
+with_monotouch_watch_default=no
+with_profile4_x_default=no
```

Comment:
This is a test for github required comments

see https://help.github.com/articles/about-pull-request-reviews/